wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -81,11 +81,11 @@
    * Contains an identifier of an editor that should be used in a workspace. Should be set/read from
    * {@link WorkspaceConfig#getAttributes}.
    *
-   * <p>Value is colon separated id, version.
+   * <p>Value is plugin id.
    *
    * <p>This is beta constant that is subject to change or removal.
    *
-   * <p>Example of the attribute value: 'org.eclipse.che.super-editor:0.0.1'
+   * <p>Example of the attribute value: 'eclipse/super-editor/0.0.1'
```

Comment:
Nitpick: I would keep the `beta` line at the bottom.

Reply:
changed